src/bin/tikv-server.rs

```diff
@@ -571,6 +571,14 @@ fn build_cfg(matches: &Matches,
                   "server.end-point-concurrency") {
         cfg.end_point_concurrency = adjust_end_points_by_cpu_num(total_cpu_num);
     }
+    cfg_usize(&mut cfg.end_point_txn_concurrency_on_busy,
+              config,
+              "server.end-point-txn-concurrency-on-busy");
+    assert!(cfg.end_point_txn_concurrency_on_busy <= cfg.end_point_concurrency &&
```

Comment:
Assert in those places should be refactored in the future. `tikv-server` is not supposed to panic before the server is started.